pkg/command/exec/exec.go

```diff
@@ -54,16 +54,6 @@ func combinedOutput(ctx context.Context, cmd *exec.Cmd, filters []string, scoped
 	return out, err
 }
 
-// CombinedOutput runs the command 'cmd' which was initialized with reference
-// to 'ctx', and logs an error to 'scopedLog', with more verbosity if 'verbose'
-// is set to true.
-//
-// Returns any error (including timeout) that may have occurred during
-// execution of 'cmd'.
-func CombinedOutput(ctx context.Context, cmd *exec.Cmd, scopedLog *logrus.Entry, verbose bool) ([]byte, error) {
```

Comment:
This function is not used anywhere. Which is probably good - that means that `CombinedOutput` method of `Cmd` struct is used everywhere.

Reply:
@joestringer Do you agree with removing it?